### DISCUSSION TITLE
Why don't I need ffmpeg?

### DISCUSSION THREAD

**Discussion by 9mean2** [k-yle/rtsp-relay#251] [Q&A]
In general, I understand that in order to display rtsp on the web, it must be encoded with ffmpeg and displayed as hls.

Can you explain the logic?

**Comment by k-yle** [comment]
displaying an RTSP stream in a browser is impractical so we need to convert the stream in a format that we can decode in the browser. That's what we use ffmpeg for—to convert an RTSP stream into an MPEG1 stream.
